src/utils/isObject.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Returns true if the given value is a plain Object.
+ */
+export function isObject(value: any): value is Record<string, any> {
```

Comment:
Using `unknown` is more suitable here, but there have been some issues with adopting `unknown` for our TypeScript users in the past. I'd migrate from `any` to `unknown`as a separate, carefully released change.